iteration.qmd

```diff
@@ -29,7 +29,8 @@ If you want to live life on the edge you can get the dev version with `devtools:
 :::
 
 In this chapter, we'll focus on tools provided by dplyr and purrr, both core members of the tidyverse.
-You've seen dplyr before, but purrr is new.
+You've seen dplyr before, but purrr LINK?
```

Comment:
Is there some existing convention in the book that would have you linking to <https://purrr.tidyverse.org> or similar, upon first mention?

Reply:
It's done for targets, for example.